README.md

```diff
@@ -11,11 +11,19 @@ pip install -r requirements-dev.txt
 ```
 
 ### Setup pre-commit
+Pre-commit is setup to automatically run formatters and linters when you commit. It is suggested to install pre-commit via pipx to keep it seperate from robot code dependencies. First install pipx:
+```
+pip install pipx
+```
```

Comment:
I'd rather point to the [pipx install docs](https://pipx.pypa.io/stable/installation/). A plain `pip install` is almost always the wrong thing to do.